### ISSUE TITLE
What are the primary cause(s) of experiments failing to add to the API?

### ISSUE THREAD

**Issue by cansavvy** [AlexsLemonade/rna-seq-accession-management#2]
### Context

Related issues: [#1358](https://github.com/AlexsLemonade/refinebio/issues/1358) and [#642](https://github.com/AlexsLemonade/refinebio/issues/642) of refine.bio 

@kurtwheeler wants to know what has caused ~2.5k accessions (all species) to fail to add to the API. What has made these not make it into our database?

*Notes about the current repository:*
- "previous_human.txt" is the full list of human accessions that have been tried to add to the API. 
- "previous_rat.txt" is the full list of rat accessions that have been tried to add to the API. 

### Problem/Question
What are the primary cause(s) of experiments failing to add to the API?
At this point, this is a fairly open ended question that I hope to shed more light to as I start digging into it. 

### Solutions or Next Steps

1) Obtain the list of full experiments in the API to date. 

2) Cross reference (by species) the full API experiment list with the lists of all accessions that have been tried: "previous_human.txt" and "previous_rat". Accessions that are in "previous" lists but NOT in the API are our list of "failed to add to the API"

3) Look for differences between "failed to add to API" and "successfully added to API". 

#### A not-at-all comprehensive list of potential suspects: 
* non-supported platforms
* non-supported technologies
* non-supported library strategies
* non-supported libraries
* other.....

### New Issue Checklist
 - [x] The title is short and descriptive.
 - [x] You have explained the context that led you to write this issue.
 - [x] You have reported a problem or idea.
 - [x] You have proposed a solution or next step.

**Comment by cansavvy** [comment]
@kurtwheeler please let me know if I've misunderstood anything.

**Comment by cansavvy** [comment]
After obtaining the failed survey accession IDs in #3, I've looked through some of these datasets manually. 

I've looked at about ~20 of the datasets and so far have found that nearly all (if not all) had some sort of glaring issue of why they didn't get into the API. Most have ChIP-seq samples as a subset or majority of their samples, some had RIP-seq, CLIP-seq, and one had something called "FiT-seq" - SRP064165.

Essentially my conclusion is that there is no cause for concern for these accessions that have been filtered out. All of the datasets I have looked through by hand have had pretty obvious reasons of why they should have failed.
